theano/sandbox/cuda/tests/test_opt.py

```diff
@@ -291,6 +291,27 @@ def test_local_gpu_subtensor():
     assert any([isinstance(node.op, cuda.GpuElemwise) for node in topo])
 
 
+def test_local_split():
+    """ Test that the GpuSplit op is being applied and works """
+    # Construct symbolic split
+    x = tensor.vector()
+    splits = tensor.lvector()
```

Comment:
That's strange, it worked fine for me, running on my GeForce GTX 780 Ti.
```
> nosetests -v test_opt.py
Using gpu device 0: GeForce GTX 780 Ti
...
Test that the GpuSplit op is being applied and works ... ok
...
----------------------------------------------------------------------
Ran 47 tests in 25.065s
```
If I change it to `fvector` I get
```
TypeError: ('splits must have type tensor.lvector', TensorType(float32, vector))
```
but with `ivector` it works fine - this is OK, right?  I'm committing this change, let me know that it works for you.